leaves - cloudy

```diff
@@ -0,0 +1,55 @@
+# frozen_string_literal: true
+
+def add(num1, num2)
+  return num1 + num2
+end
+
+def subtract(num1, num2)
+  return num1 - num2
+end
+
+def multiply(num1, num2)
+  return num1 * num2
+end
+
+def divide(num1, num2)
+  return num1 / num2
+end
+
+puts 'Welcome to Calculator. What operation do you want to use'
+puts "Please choose a name or a symbol from the list below:
+        1. add(+)
+        2. subtract(-)
+        3. multiply(*)
+        4. divide(/)"
+
+operation = gets.chomp.downcase
+
+valid_operations = ['add', '+', 'subtract', '-', 'multiply', '*', 'divide', '/']
+
+until valid_operations.include?(operation)
+  print 'Please enter a valid operation: '
+  operation = gets.chomp.downcase!
+end
+
+puts 'please input the first number'
+num1 = gets.chomp.to_i
```

Comment:
This allows for silly input like this: 

```
Please choose a name or a symbol from the list below:
        1. add(+)
        2. subtract(-)
        3. multiply(*)
        4. divide(/)
add
please input the first number
jfdskal;dfjlsa
please input the second number
fjdkaljfdslkajkfadls
awesome, your total is 0! whooo.
```

You need to do a ~liiittle~ more work to get this to work the way you expect.